packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/PostProcessing/GeneratedCodeWorkspace.cs

```diff
@@ -20,6 +22,7 @@ namespace Microsoft.Generator.CSharp
     internal class GeneratedCodeWorkspace
     {
         private const string GeneratedFolder = "Generated";
+        private const string InternalFolder = "Internal";
```

Comment:
so this implementation looks very specific to our implementations, should we put it in a place so that others could customize or extend on?